catalog/core/catalog-core-definitionparser/src/main/java/ddf/catalog/definition/impl/DefinitionParser.java

```diff
@@ -138,6 +141,8 @@
           .setLenient()
           .create();
 
+  public static final int FILE_POLL_INTERVAL = 997;
```

Comment:
✏️ could specify this is milliseconds in the name or in a comment

Reply:
Done